azext_edge/tests/edge/support/create_bundle_int/helpers.py

```diff
@@ -268,7 +269,13 @@ def get_file_map(
         file_map["usage"] = convert_file_names(walk_result[c_path]["files"])
         file_map["__namespaces__"]["usage"] = c_namespace
     elif ops_service == "acs":
-        assert len(walk_result) == 1 + expected_default_walk_result
+        if not acstor_namespace:
```

Comment:
switch the if branches

also comment for the magic numbers (why are you adding 2 vs 1 vs nothing)